### ISSUE TITLE
CellRanger indexes are not recognized

### ISSUE THREAD

**Issue by golharam** [clintval/sample-sheet#92]
Is it possible to bypass index validation?

```
    sample_sheet = SampleSheet(args.samplesheet)
  File "/home/ec2-user/cellranger-docker/env/lib/python3.6/site-packages/sample_sheet/__init__.py", line 418, in __init__
    self._parse(self.path)
  File "/home/ec2-user/cellranger-docker/env/lib/python3.6/site-packages/sample_sheet/__init__.py", line 524, in _parse
    self.add_sample(Sample(dict(zip(sample_header, line))))
  File "/home/ec2-user/cellranger-docker/env/lib/python3.6/site-packages/sample_sheet/__init__.py", line 296, in __init__
    raise ValueError(f'Not a valid index: {value}')
ValueError: Not a valid index: SI-GA-F8
```

**Comment by clintval** [comment]
Looks reasonable to me! We could update what is considered a valid index to include the union of characters Illumina and 10x allow here:

https://github.com/clintval/sample-sheet/blob/8da5c6b5ef69170c456b6e7b7673fc2d57536fa2/sample_sheet/__init__.py#L275

I probably won't have time to get to in the next few days FYI but it's now on my radar.

**Comment by golharam** [comment]
I just submitted a PR with the proposed change.

**Comment by clintval** [comment]
Thank you for the patch, and the unit test! I just published `sample_sheet==0.10.0` with this feature. Please pip install! Bioconda will find the change automatically in the next few days.

**Comment by clintval** [comment]
I'm migrating to an official public API freeze in `1.0.0`. The API may change slightly, but function should remain the same.

Any feature requests, critiques of the API, and criticisms welcome before I commit. Thanks!

**Comment by golharam** [comment]
I just pushed a final change as I missed an index set in the first PR.   I wonder if an option to bypass index checking would be good for a future release.

**Comment by clintval** [comment]
Thanks for the idea! Noted!

**Comment by clintval** [comment]
Just published `0.11.0` to PyPi, Bioconda should follow after some period of lag.
